playback/tape_recorder.py

```diff
@@ -148,11 +148,11 @@ def _add_post_operation_metadata(recording, metadata, post_operation_metadata_ex
         """
         metadata[TapeRecorder.DURATION] = duration
         metadata[TapeRecorder.RECORDED_AT] = str(datetime.utcnow())
-        outputs = TapeRecorder._extract_recorded_output(recording, direct_access=True)
+        output_keys = TapeRecorder._extract_recorded_output_keys(recording)
```

Comment:
If you are putting it in the same commit (or even if you split it to another one), please add to the commit message why we changed from get_data to get_all_keys for future reader